### PR TITLE
fix scheduler when not supplied

### PR DESCRIPTION
**Pull request by gshennvm**
# What does this PR do ?

when user does not supply learning rate scheduler, nemo aligner will crash. but now we use a dummy scheduler object

### CODE REVIEW

File: nemo_aligner/utils/train_script_utils.py [hunk 1]
```diff
@@ -124,15 +134,15 @@ def add_custom_checkpoint_callback(ptl_trainer, ptl_model):
...
-        def custom_save(self, *args, **kwargs):
-            ...
-
     return FakeCheckpointCallback()
 
 
 def extract_optimizer_scheduler_from_ptl_model(ptl_model):
-    return ptl_model.optimizers().optimizer, ptl_model.lr_schedulers()
+    scheduler = ptl_model.lr_schedulers()
+    if scheduler is None:
```
[the remark this answers is not in the document]
Comment (gshennvm) [reply nemo_aligner/utils/train_script_utils.py:146]: yup! 31ab380170dd1f679d1a513852bf84cd2c93f39c

File: nemo_aligner/utils/train_script_utils.py [hunk 2]
```diff
@@ -104,9 +104,9 @@ def init_using_ptl(ptl_trainer, ptl_model, train_dataloader, train_ds):
     ptl_trainer._checkpoint_connector._restore_modules_and_callbacks(ptl_trainer.ckpt_path)
     ptl_trainer._checkpoint_connector.restore_training_state()
     ptl_trainer._checkpoint_connector.resume_end()
-    if ptl_model._scheduler is not None:
-        scheduler = ptl_model._scheduler["scheduler"]
+    scheduler = ptl_model.lr_schedulers()
```
[the remark this answers is not in the document]
Comment (gshennvm) [reply nemo_aligner/utils/train_script_utils.py]: 31ab380170dd1f679d1a513852bf84cd2c93f39c

**Comment by gshennvm** [comment]
> Just some small suggestion to make everything consistent.
> 
> In addition to the comments below, we should also remove the `if self.scheduler is not None:` in supervised.py

done!

**Review by odelalleau — APPROVED** [review]
Looks good, thanks!
